src/main/java/com/nessynet/yaacs/configuration/DynamoDBConfiguration.java

```diff
@@ -0,0 +1,91 @@
+package com.nessynet.yaacs.configuration;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.annotation.PostConstruct;
+
+import com.amazonaws.auth.AWSCredentials;
+import com.amazonaws.auth.AWSCredentialsProvider;
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.client.builder.AwsClientBuilder.EndpointConfiguration;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClientBuilder;
+import com.amazonaws.services.dynamodbv2.model.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```

Comment:
Are we using sl4j as our logging interface and then allowing it to bridge over to log4j internally?